Assets/Scripts/Player/PlayerItemPickup/MousePickup.cs

```diff
@@ -0,0 +1,40 @@
+using UnityEngine;
```

Comment:
Same here, anything preventing this from being condensed into MouseInventoryManager?